src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractImportCompletionProvider.cs

```diff
@@ -97,6 +96,40 @@ private async Task<HashSet<string>> GetNamespacesInScopeAsync(SyntaxContext synt
             return namespacesInScope;
         }
 
+        private static ImmutableArray<string> GetImportedNamespaces(SyntaxContext context, CancellationToken cancellationToken)
+        {
+            var position = context.Position;
+
+            var trivia = context.SyntaxTree.FindTriviaAndAdjustForEndOfFile(position, cancellationToken, findInsideTrivia: true);
+            var triviaToken = trivia.Token;
+
+            // If we are inside of leading trivia of a token adjust position to be the start of that token.
+            // This is a workaround for an issue, when immediately after a `using` directive it is not included into the import scope.
+            // See https://github.com/dotnet/roslyn/issues/67447 for more info.
+            if (triviaToken.SpanStart > context.Position)
+                position = triviaToken.SpanStart;
```

Comment:
For a workaround, I'd prefer to limit the adjustment to case known to be problematic instead, i.e. in global statement context after using. Maybe extract a helper from [here](https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/Recommendations/CSharpRecommendationServiceRunner.cs,4f32bcb9b50c7795,references) and use that.